pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -378,7 +378,7 @@ func getMaxVolumeFunc(filterName string) func(node *v1.Node) int {
 }
 
 func getMaxEBSVolume(nodeInstanceType string) int {
-	if ok, _ := regexp.MatchString("^[cm]5.*", nodeInstanceType); ok {
+	if ok, _ := regexp.MatchString("^[cmr]5.*|t3|z1d", nodeInstanceType); ok {
```

Comment:
Having the same regex in two places that need to be kept in sync makes me uneasy. Also, for someone who is not in the know, how would I learn whether this regex is correct? Are there test cases for the supported AWS volume types?

Reply:
There is an existing test from the previous PR #66397 for m5/c5. Updated commit description and title. Thanks!